foodsaving/subscriptions/management/commands/cleanup_channel_subscriptions.py

```diff
@@ -0,0 +1,17 @@
+from dateutil.relativedelta import relativedelta
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from foodsaving.subscriptions.models import ChannelSubscription
+
+
+class Command(BaseCommand):
+    """If we have not received a message on the channel for a while, we delete our entries."""
+
+    def handle(self, *args, **options):
+        def print(*args):
+            self.stdout.write(' '.join([str(_) for _ in args]))
+
+        ChannelSubscription.objects.filter(lastseen_at__lt=timezone.now() - relativedelta(minutes=5)).delete()
+
+        print('done')
```

Comment:
This makes the test output a tad more confusing:

```
............................................................................................................................................................................................done
.done
.................................................................
----------------------------------------------------------------------
Ran 254 tests in 12.275s
```

I would either get rid of the 'done' or redirect it during testing.